docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md

```diff
@@ -0,0 +1,130 @@
+
+For more information on authentication and security in Amazon Bedrock, refer to
+the [Amazon Bedrock security and privacy](https://aws.amazon.com/bedrock/security-compliance/).
+
+## Region
+
+In the **Region** field write the region of the deployed endpoint.
+
+## Action
+
+There are two possible actions with the amazon bedrock connector
+
+### Invoke Model
+
+This action is meant to invoke a model with a raw payload.
+
+A model id has to be specified, you can find all the available one for amazon
+bedrock [here](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html)
+Be sure the model is available on your region, that your model can invoke the `Invoke Model` action, and you are using a user with adequate rights.
+
+The payload is dependent on the model used, you can find all different
+payload [here](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html)
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables.
+
+The response is dependent on the model used, you can find all returned
+response [here](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html)
+
+#### Example
+
+If using the model `Jamba-instruct` with modelId `ai21.jamba-instruct-v1:0`.
+
+Looking at the [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-jamba.html)
+
+The payload could be:
+
+```json
+{
+  "messages": [{ "role": "user", "content": "Hello" }],
+  "max_tokens": 256,
+  "top_p": 0.8,
+  "temperature": 0.7
+}
+```
+
+and the FEEL mapping could be:
+
+```
+{ response : body.choices.message.content[1] }
+```
+
+### Converse
+
+This action is meant to start or continue a conversation with a model.
+
+A model id has to be specified, you can find all the available one for amazon
+bedrock [here](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html)
+Be sure the model is available on your region, that your model can invoke the `Converse` action, and you are using a user with adequate rights.
+
+`New Message` is either the first message (to start a conversation) or is the next message from an already started conversation
+`Message History` is the history of the conversation that should always be passed. If not set, this will be a new conversation
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables.
+
+The response contains two elements:
+
+`messageHistory` is the full history of previous message, from user and assistant, including the latest message written by the assistant
+`newMessage` is the latest message written by the assistant
+
+Ideally, the messages history needs to transit within the process and needs to be the input of this `Converse` task.
+With the new message
```

Comment:
I think this was either carried down a line or an incomplete sentence, but made an adjustment in my latest commit.